tests/release/pipelines/README.md

```diff
@@ -1,41 +1,44 @@
-# Release Pipelines Tests
+# Release pipelines e2e tests suite
 
-This suite contains e2e-tests for testing release pipelines from repository [release-service-catalog](https://github.com/redhat-appstudio/release-service-catalog/tree/main), those tests run against RHTAP RH-Stage.
+This suite contains e2e-tests for testing release pipelines from repository [release-service-catalog](https://github.com/redhat-appstudio/release-service-catalog/tree/main).
```

Comment:
the default branch is development now, not main. Can you update it?

Reply:
Updated